ai21/models/chat/__init__.py

```diff
@@ -0,0 +1,5 @@
+from __future__ import annotations
+
+from .chat_message import ChatMessage as ChatMessage
```

Comment:
Why do you need these aliases?

Reply:
`ruff` was removed imports that weren't in the main `__init__.py` saying these imports were not used, where in-fact they were. This was a workaround I saw was used. 

I found a better solution to configure ruff to not removed these types of imports from any `__init__.py` (see `pyproject.yaml`